.vsts.pipelines/jobs/ci-linux.yml

```diff
@@ -70,6 +72,16 @@ jobs:
     displayName: Build source-build
     timeoutInMinutes: 120
 
+  # Generate prebuilt burndown data
+  - script: |
+      set -x
+      df -h
+      $(docker.run) $(docker.src.map) $(docker.src.work) $(imageName) ./build.sh \
+        /t:GeneratePrebuiltBurndownData
```

Comment:
This will always just generate the online data, right? I don't think there's a way `OfflineBuild` will be set here.

I think this target should be changed to just generate both the online and offline data since you don't need build outputs to do it.